templates/about/release-cycle.html

```diff
@@ -578,15 +578,15 @@ <h2 id="ubuntu-openstack-release-cycle">
     </div>
 
     <div class="u-fixed-width">
-      <p>For more information on previous and upcoming Ubuntu OpenStack releases please see the <a href="https://wiki.ubuntu.com/OpenStack/CloudArchive " class="p-link--external ">Ubuntu Cloud Archive wiki page</a>.</p>
+      <p>For more information on supported versions, see <a href="https://ubuntu.com/openstack/docs/supported-versions" class="p-link--external ">Charmed OpenStack documentation</a>.</p>
```

Comment:
This is no longer an external link, so please change it to relative path `/openstack/docs/supported-versions` and remove the `class="p-link--external "`

Reply:
Updated, should work now